db/seeders/20220918075630-seed-data.js

```diff
@@ -1,24 +1,57 @@
-'use strict';
-
 module.exports = {
-  async up (queryInterface, Sequelize) {
-    /**
-     * Add seed commands here.
-     *
-     * Example:
-     * await queryInterface.bulkInsert('People', [{
-     *   name: 'John Doe',
-     *   isBetaMember: false
-     * }], {});
-    */
+  up: async (queryInterface) => {
+    // Define user data
+    const userData = [
+      {
+        first_name: 'John',
+        last_name: 'Doe',
+        email: 'john@gmail.com',
+        password: 'cec96aa1b911244e1657b54aadb408d60dcc742b7b3eede48e2fd7708589b048c6f7ac3fa6667e002f9ec80ec4d8d43fe24982a87ffe7f9f72be33fae6e3590c',
```

Comment:
Do we know what these unHashed values are? Do we want to use a SALT value as well ? you can define it in the .env file

Reply:
I agree, we should probably keep the unhashed value somewhere. it might be hard for us to log in to the seed users later if we don't write the password down 🤷‍♂️

Reply:
hahah yes, sorry, got lazy for a moment and just plonked the hashed version of 'password' here. Will update.